VR_RV/ProjectSettings/TagManager.asset

```diff
@@ -17,8 +17,13 @@ TagManager:
   - UI
   - Vacuum
   - Sphere
+<<<<<<< peang_pickup
```

Comment:
Should these merge conflict lines be removed?

Reply:
Yes